docs/changelog/1928.feature.rst

```diff
@@ -0,0 +1 @@
+Implemented [] substitution (alias for {posargs})
```

Comment:
quote both ``[]`` and ``{posargs}`` also add your user name as others do.

Reply:
done